client/post-editor/post-editor.jsx

```diff
@@ -451,7 +492,12 @@ var PostEditor = React.createClass( {
 			content: autosave.content
 		};
 
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
 		actions.edit( edits );
+
+		this.props.setTitle( autosave.title );
+		this.props.setExcerpt( autosave.excerpt );
+		this.props.setContent( autosave.content );
```

Comment:
Do you envision that we will have some sort of facility to set all of these things in one API call? That's one issue with breaking up all of these little setters, it makes it harder to do a number of things at once.